pages/index.js

```diff
@@ -1,3 +1,21 @@
+import useSWR from "swr";
+import EventCard from "../components/EventCard";
+import { fetcher } from "../helpers/api";
+
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  const { data: events, error } = useSWR("/api/events", fetcher);
+  if (error) return <h1>There was an error</h1>;
+  if (!events) return <h1>...Loading...</h1>;
+
+  events.sort((a, b) => new Date(a.date) - new Date(b.date));
```

Comment:
Please keep in mind, that you are mutating the original array here. This is not, what we want 🙈 
```suggestion
  const sortedEvents = events.sort((a, b) => new Date(a.date) - new Date(b.date));
```

Reply:
but wouldn't I then have to do something like `const sortedEvents = [...events].sort...` so the original array won't be mutated

Reply:
🤦🏼‍♀️ Sorry! You are right, of course. 🙂